pocs/scheduler/constraint.py

```diff
@@ -146,11 +107,11 @@ def get_score(self, time, observer, observation, **kwargs):
         except KeyError:
             self.logger.error("Moon must be set")
 
-        moon_sep = moon.separation(observation.field.coord).value
+        moon_sep = observation.field.coord.separation(moon).value
 
         # This would potentially be within image
         if moon_sep < 15:
-            self.logger.debug("Moon separation: {}".format(moon_sep))
+            self.logger.debug("\t\tMoon separation: {:.02f}".format(moon_sep))
```

Comment:
`\t\t` seems unnecessary.

Reply:
It helps with the output in this case and makes it consistent with the other scheduler items. If we don't want here we should remove everywhere.

Reply:
Remove everywhere. I've been guilty of making these changes in the past, and they stay long after they are useful. Perhaps it would be better to use a custom log colorizer.

Reply:
Agreed about the tabs but I do think they are useful here. Long ago I had them all over the logs and it became a nightmare. Then I started thinking about how to write some logger wrapper that would auto-figure out the indent. Luckily I wisely abandoned the whole idea and removed all indents...except for here in the scheduler call.

I know @AnthonyHorton has used some in the SBIG and FLI code and I've been meaning to mention removing them. ;)

I'm going to leave in here unless very serious concerns (see notebook for how output looks) and again the additions here were to make the whole scheduler output consistent as it otherwise looked pretty bad. 

Agree that we should try to avoid them elsewhere (i.e I get to be the exception but no one else. 😁).

Reply:
I think you see the slippery slope all to well!